### PR TITLE
Niko/authenticated calls

### PR DESCRIPTION
**Pull request by niko79542**
+ Authenticated calls
+ get follower post query
+ get hundredx query 

This is the 5th and final pr in the list of broken up pr's...

### CODE REVIEW

File: src/services/postService.js [hunk 1]
```diff
@@ -4,8 +4,12 @@ const UserModel    = db.sequelize.models.user;
 const Op           = db.Sequelize.Op;
 const _            = require('lodash');
 const ReplyService = require('./replyService');
+const client       = require('./steem');
 const {
```
Comment (jay-ithiel) [inline src/services/postService.js:8]: Are we using steem client now? 
I think they offer a way to fetch posts in html format via their api. They have an example in their docs if I recall correctly. Might not even have to use Remarkable

File: src/services/postService.js [hunk 2]
```diff
@@ -100,4 +136,19 @@ module.exports = {
...
+    ))
+  },
+  findByPermLinkAndAuthor: function(permLink, name) {
+    return PostModel.findOne({
+      where: { permLink },
+      include: [
+        { model: UserModel,
+          where: { name }
+        }
+      ]
```
Comment (jay-ithiel) [inline src/services/postService.js:151]: this looks weird. It's not a big deal but could we make it
```
...
include: [
   {
      model: UserModel,
      where: { name }
   }
]
...
```

File: src/services/userAuthentication.js [hunk 3]
```diff
@@ -28,22 +24,67 @@ class UserAuthentication {
         throw err
```
Comment (jay-ithiel) [inline src/services/userAuthentication.js:24]: We should do
```
throw new Error(`Custom error message: ${err}`)
```

File: src/services/userAuthentication.js [hunk 4]
```diff
@@ -28,22 +24,67 @@ class UserAuthentication {
...
 
+  initializeSteemUser(accessToken) {
+    return this.steemUser = sc2.Initialize({
+      app: 'hundredx.app',
+      callbackURL: process.env.DEV_URL,
+      accessToken,
+    });
+  }
+
+  getUsersFollowerPosts() {
```
Comment (jay-ithiel) [inline src/services/userAuthentication.js:76]: This name is a bit confusing. Not sure what it's fetching even after reading the code haha

File: src/services/postService.js (hunk 2 above)
Comment (niko79542) [reply src/services/postService.js:151]: sure

File: src/services/userAuthentication.js (hunk 3 above)
Comment (niko79542) [reply src/services/userAuthentication.js:24]: will make this change

File: src/services/userAuthentication.js (hunk 4 above)
Comment (niko79542) [reply src/services/userAuthentication.js:76]: how bout getPostsOfMyFollowers

Comment (jay-ithiel) [reply src/services/userAuthentication.js:76]: yeah that works

File: src/services/postService.js (hunk 1 above)
Comment (niko79542) [reply src/services/postService.js:8]: This has been addressed with Remarkable.

**Comment by niko79542** [comment]
Merged.  Closing

**Comment by jay-ithiel** [comment]
Same with this. New commits haven't been pushed

**Comment by niko79542** [comment]
Hey - these have actually been pushed.  Rebasing wasnt too easy, so I just cherry picked the commits from here and added in changes from your comments.  re-closing.

**Comment by jay-ithiel** [comment]
got it 👍

**Pull request closed without merging**